devwilliamy/coverland
# New Seat Cover Reviews Added

https://app.clickup.com/t/8688mzatw

## To Test
- Check reviews trigger still brings up review window  (under the product title)
- Ensure review section still works
- See more review images works and clicking into them and around
- sort and filter work
- sort nad filter behavior persist on whichever htey're on
- view more works as intended and has the sort/filter applied
- check it works on mobile 
- check both car cover & seat cover
- check make/model/year pages

Things to note - 
- if it says there are reviews but no reviews show up, let me know what page you were on -- since our reviews are huge now it'll timeout and i need to know examples to optimize database
- we are default sorting to SKU & highest helpful count
- if we've seen an image before it won't show up again
